ts/features/idpay/initiative/configuration/xstate/machine.ts

```diff
@@ -375,7 +401,11 @@ const createIDPayInitiativeConfigurationMachine = () =>
         })),
         confirmIbanOnboarding: assign((_, event) => ({
           ibanBody: event.ibanBody
-        }))
+        })),
+        showInstrumentsLoadingErrorToast: () =>
+          showToast("Errore nel caricamento degli strumenti", "danger"),
+        showIbanLoadingErrorToast: () =>
+          showToast("Errore nel caricamento degli IBAN", "danger")
```

Comment:
What do you think about a generic function to handle error toasts instead? 

```suggestion
        showErrorToast: (message: string) => showError(message, "danger"),
```

Also, we need to use localized messages here.

Reply:
a generic function would lead to duplication, forcing all the states that need to call it to basically call `showToast` with the same message